src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const mass = sourceString.split('\n');
+  const mass2 = mass.join('').split(';');
+  const res = mass2.map(item => item.split(':'));
```

Comment:
```suggestion
  const res = sourceString.split('\n')
    .join('').split(';')
    .map(item => item.split(':'));
```
Можна було б це все записати ланцюжком